python/pylibwholegraph/CMakeLists.txt

```diff
@@ -127,9 +127,6 @@ message(VERBOSE "PYLIBWHOLEGRAPH: Enable detection of conda environment for depe
 ##############################################################################
 # - Compiler options ---------------------------------------------------------
 
-# this is needed for clang-tidy runs
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
-
```

Comment:
This is just a cleanup, not a behavior change... this option is already set to `ON` earlier in the same file.

https://github.com/rapidsai/cugraph-gnn/blob/71675d868589ff9f904197f729985de1555cb914/python/pylibwholegraph/CMakeLists.txt#L43